apps/web/pages/org/[orgSlug]/[user]/[type].tsx

```diff
@@ -55,8 +55,8 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
 type Props = UserTypePageProps | TeamTypePageProps;
 
 export default function Page(props: Props) {
-  if ((props as TeamTypePageProps)?.teamId) return TeamTypePage(props as TeamTypePageProps);
```

Comment:
Converting these function calls to JSX fixes the issue because with function calls we end up with wrong hooks order.